src/components/InputEditor.tsx

```diff
@@ -18,6 +18,34 @@ import type { ParserResult, TransformationResult } from '../swc'
 
 const STORAGE_KEY = 'v1.code'
 
+function getReportIssueUrl({
+  inputCode,
+  version,
+  config,
+  playgroundLink,
+}: {
+  inputCode: string
+  version: string
+  config: string
+  playgroundLink: string
+}): string {
+  let reportUrl = `https://github.com/swc-project/swc/issues/new?assignees=&labels=C-bug&template=bug_report.yml`
+
+  const inputCodeMarkdown = encodeURIComponent(
+    '```tsx\n' + inputCode + '\n```\n'
+  )
+  reportUrl += `&code=${inputCodeMarkdown}`
+
+  const configMarkdown = encodeURIComponent('```json\n' + config + '\n```\n')
+  reportUrl += `&config=${configMarkdown}`
+
+  reportUrl += `&repro-link=${encodeURIComponent(playgroundLink)}`
+
+  reportUrl += `&version=${version}`
```

Comment:
It would be better that using `URLSearchParams` instead of concatenating strings manually.